ncj/vray/render-linux-with-blobfuse-mount/installblobfuse.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
Cool. At this stage there is a bug in blobfuse which I have reported here: https://github.com/Azure/azure-storage-fuse/issues/180 : if we will use `set e` it bubbles up the warning and stops, (reason: `the blobfuse.xxx.rpm pkg is not signed`)

Reply:
:bulb: note: **Or** for blobfuse signed issue, I can add `--nogpgcheck` but that will beat the extended purpose of `set e`, although at latter stage which should be soon once blobfuse guys have released a new version hopefully this will be resolved.